.github/workflows/fossa.yml

```diff
@@ -15,4 +15,3 @@ jobs:
       - uses: fossas/fossa-action@v1.4.0
         with:
           api-key: ${{ secrets.FOSSA_API_KEY }}
-
```

Comment:
any reason to remove all the newlines at the end? not having them can cause some "issues" on git diff

Reply:
It is an popular hook that removes EOF, And I have added this since I have used it a lot

Reply:
See https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#end-of-file-fixer and https://github.com/pre-commit/pre-commit-hooks/blob/main/pre_commit_hooks/end_of_file_fixer.py

Reply:
I saw that we have extra new line on some files, and not having same rule in every file is not a great idea I think

Reply:
Yes it'd be good to be consistent throughout.
If we keep this validation then let's also include the rawfile/csi/ as well.

Reply:
I have excluded them since they where some generated files

Reply:
could we exclude those ones generated only?
anyway, this is fine lgtm

Reply:
Yeah, I has support for globing also